modules/vpc/src/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  identifier = replace(lower(var.identifier), "/[^a-z1-9]/", "")
+
+  short_regions = {
```

Comment:
I think this would limit the module to only these two regions, and fail otherwise.

Default on the lookup short term?  [This sorta thing](https://github.com/cloudposse/terraform-aws-utils/blob/main/main.tf)?

Reply:
Dropped.